guides/common/modules/proc_installing-the-satellite-server-packages.adoc

```diff
@@ -32,7 +32,7 @@ include::proc_installing-server-packages-el.adoc[]
 
 endif::[]
 
-ifdef::foreman-el,katello[]
+ifdef::foreman-el,katello,satellite[]
 == [[rhel-7]]Red Hat Enterprise Linux 7
```

Comment:
This is a valid hunk.

Reply:
Oh was this merged already: https://github.com/theforeman/foreman-documentation/pull/722